### DISCUSSION TITLE
Русский перевод(Russian translation)

### DISCUSSION THREAD

**Discussion by Drakosha405** [Acly/krita-ai-diffusion#991] [General]
Добавляю файл с русской переводом дополнения. 
Если вы заметили ошибки или неточности прошу писать сюда или самим вносить изменения с последующим добавлением новых версий файла.
некоторые места я оставил без перевода так как считаю что перевод этих слов не будет понятен или люди будут долго искать эти параметры следуя урокам из интернета.
Заранее извиняюсь если будут ошибки пытался не просто перевести но и местами адаптировать.

Прошу не стесняться вносить предложения как можно улучшить перевод.

I am adding a file with a Russian translation.
also here I ask people to test it for errors or improvements.
[ru.json](https://github.com/user-attachments/files/16403065/ru.json)

**Comment by Drakosha405** [comment]
@Acly
![image](https://github.com/user-attachments/assets/d957c18d-2143-41c7-a587-65b44b70cc0c)
in the performance settings window, I would like to move the long text to a new line, but "\\n" does not work here.

![image](https://github.com/user-attachments/assets/4360b65d-da3e-4ffb-a6df-fafe33343366)
there is no option to translate "Batches" in the queue/cancel window

**Reply by Acly** [reply]
> in the performance settings window, I would like to move the long text to a new line, but "\n" does not work here.

`<br>` might work

> there is no option to translate "Batches" in the queue/cancel window

I added it recently
